.ibm/tools/tests-results/go.mod

```diff
@@ -0,0 +1,29 @@
+module github.com/feloy/junit-collector
```

Comment:
Can we change the module to something like `github.com/redhat-developer/junit-collector`?